apps/paf/fields/index.js

```diff
@@ -19,6 +21,8 @@ const moment = require('moment');
 const after1900Validator = { type: 'after', arguments: ['1900'] };
 const PRETTY_DATE_FORMAT = 'Do MMMM YYYY';
 
+const currentYear = new Date().getFullYear().toString();
+const afterCurrentYearValidator = { type: 'after', arguments: [currentYear] };
```

Comment:
This is the right way to validate, but at the moment if you give only '2024' as an argument to the validator it will check for any dates after 01-01-2024. That means any date between the start of this year and today will be accepted. 

If you want to be strict about the date being in the future I would re-work it so that you pass the validator today's full date instead of only the year.

Reply:
> This is the right way to validate, but at the moment if you give only '2024' as an argument to the validator it will check for any dates after 01-01-2024. That means any date between the start of this year and today will be accepted.
> 
> If you want to be strict about the date being in the future I would re-work it so that you pass the validator today's full date instead of only the year.

That is fixed now, rather than passing only the year using  'const currentYear = new Date().getFullYear().toString(); ' , i have. just passed the whole current date and it worked